src/pages/ContractInfoPage/components/ContractEditing.vue

```diff
@@ -73,24 +78,32 @@ const props = defineProps<{
   methodToEdit: ContractEditingType
 }>()
 
+const { t } = useI18n()
 const route = useRoute()
 const web3ProvidersStore = useWeb3ProvidersStore()
 
-const isSubmitting = ref(false)
-const isSubmitted = ref(false)
+const isLoaded = ref(true)
 
 const form = reactive(
-  props.methodToEdit.inputs.reduce((acc, _, idx) => {
-    acc[`input-${idx}`] = ''
-    return acc
-  }, {}),
+  props.methodToEdit?.inputs
+    ? props.methodToEdit?.inputs.reduce((acc, _, idx) => {
+        acc[`input-${idx}`] = ''
+        return acc
+      }, {})
+    : {},
 )
```

Comment:
You switched to `input-${id}` pattern, so probably you shouldn't use `idx` here.

Reply:
done